styles.css

```diff
@@ -1,3 +1,3 @@
 .color_text {
```

Comment:
![image](https://user-images.githubusercontent.com/74744787/158467957-d757862e-3187-47dd-beeb-dddc96aaacfd.png)

Pull Request should have a summary describing introduced changes in more detail, this pull request is supposed to describe all the changes you did in this project

Reply:
@ABDULALI3468 yeah  please just change the title of the pull request and the PR description and then request a new review